appmap/test/test_django.py

```diff
@@ -116,6 +121,21 @@ def test_unnamed(client, events):
     # message attribute.
     assert len(events[0].message) == 0
 
+def test_exception(client, events):
+    try:
```

Comment:
You should use `pytest.raises` to verify that the expected exception is being raised: https://docs.pytest.org/en/reorganize-docs/new-docs/user/pytest_raises.html .